src/core/js/app.js

```diff
@@ -37,11 +37,16 @@ require([
     $('#wrapper').append(template());
 
     // Create config model
-    Adapt.config = new ConfigModel(null, {url: "course/config.json", reset:true});
+    Adapt.config = new ConfigModel(null, { url: 'course/config.json', reset: true }).on({
```

Comment:
This looks a little bit messy all on one line - sorry but would you mind changing to
```
Adapt.config = new ConfigModel(null, {url: "course/config.json", reset:true});
Adapt.config.on({
    'change:_activeLanguage': onLanguageChange,
    'change:_defaultDirection': onDirectionChange
});
```
?